src/components/PostDetails.tsx

```diff
@@ -1,117 +1,120 @@
-import React from 'react';
-import { Loader } from './Loader';
+import React, {
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import { Post } from '../types/Post';
+import { deleteComment, getComments } from '../utils/loadutil';
+import { StateContext } from './AppContext';
+import { ACTIONS } from '../utils/enums';
 import { NewCommentForm } from './NewCommentForm';
+import { Comment } from '../types/Comment';
+import { Loader } from './Loader';
+
+type Props = {
+  post: Post,
+};
+
+export const PostDetails: React.FC<Props> = ({ post }) => {
+  const { state, dispatch } = useContext(StateContext);
+  const [showNewCommentForm, setShowNewCommentForm] = useState(false);
```

Comment:
Rename boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isShowNewCommentForm, setIsShowNewCommentForm] = useState(false);
```